Drop-In/src/main/java/com/braintreepayments/api/DropInLifecycleObserver.java

```diff
@@ -0,0 +1,46 @@
+package com.braintreepayments.api;
+
+import androidx.activity.result.ActivityResultCallback;
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.ActivityResultRegistry;
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.lifecycle.Lifecycle;
+import androidx.lifecycle.LifecycleEventObserver;
+import androidx.lifecycle.LifecycleOwner;
+
+// NEXT_MAJOR_VERSION: Update to implement DefaultLifeCycleObserver when Java 7 support is explicitly dropped.
+public class DropInLifecycleObserver implements LifecycleEventObserver {
```

Comment:
We can go ahead and just use `DefaultLifeCycleObserver` in Drop-in since we dropped Java 7 with the major version